plugin/dev/src/main/resources/application.yaml

```diff
@@ -0,0 +1,191 @@
+debug: true
+
```

Comment:
I don't think you need all this configuration in the plugin.  The plugin should only specify the stuff specific to it.